src/Model/Behavior/SequenceBehavior.php

```diff
@@ -259,6 +246,89 @@ public function beforeDelete(Event $event, Entity $entity)
     }
 
     /**
+     * Decrease the position of the entity on the list
+     *
+     * If a "higher" entity exists, this will also swap positions with it
+     *
+     * @param \Cake\Datasource\EntityInterface $entity The entity that is going to be saved.
+     * @return bool
+     */
+    public function moveUp(EntityInterface $entity)
```

Comment:
Then I'd have to worry about shifting more entities around, is that fine by you?